src/Type.hs

```diff
@@ -98,22 +100,26 @@ instance ShortShow a => ShortShow (TyF a) where
 
 
 class TyLike a arg | a -> arg where
-  tSyntax    :: a
-  tInteger   :: a
-  tString    :: a
-  tFun1      :: arg -> arg -> a
-  tMacro     :: arg -> a
-  tType      :: a
-  tDatatype  :: Datatype -> [arg] -> a
-  tSchemaVar :: Natural -> a
-  tMetaVar   :: MetaPtr -> a
+  tSyntax     :: a
+  tInteger    :: a
+  tString     :: a
+  tOutputPort :: a
+  tFun1       :: arg -> arg -> a
+  tMacro      :: arg -> a
+  tIO         :: arg -> a
+  tType       :: a
+  tDatatype   :: Datatype -> [arg] -> a
+  tSchemaVar  :: Natural -> a
+  tMetaVar    :: MetaPtr -> a
```

Comment:
Ah, this is why I complain about aligned source code... It's not easy to glance at this and tell what's changed! :grumpy: :opinionated:

Reply:
A shame we can't tell the web interface to use a better diff algorithm.

Reply:
try clicking on the gear icon and checking "hide whitespace changes", it makes it very clear which of these lines have changed ;)